h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -383,6 +387,18 @@ private RowData reconstructedToRowData(double[] reconstructed) {
     return result;
   }
 
+  private double calculateReconstructionError(double [] original, double [] reconstructed, double [] normmul, int numNumeric){
```

Comment:
Maybe we can move this function to DeeplearningMojoModel? Then you would not need to access `mojoModel._normmul` and `mojoModel._nums`. This would also expose the functionality to more users.

Reply:
Yes, it is a better solution and I like it.

I was just inspired by code in the wrapper, where similar issues are solved by this way... I will look at it...